Content.Server/StationGoal/StationGoalPaperSystem.cs

```diff
@@ -0,0 +1,120 @@
+using System.Data;
+using System.Text.RegularExpressions;
+using Content.Server.GameTicking;
+using Content.Server.Fax;
+using Content.Server.Station.Systems;
+using Content.Shared.CCVar;
+using Content.Shared.Random;
+using Content.Shared.Random.Helpers;
+using Robust.Shared.Configuration;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.StationGoal;
+
+/// <summary>
+///     System for station goals
+/// </summary>
+public sealed class StationGoalPaperSystem : EntitySystem
+{
+    [Dependency] private readonly IPrototypeManager _prototype = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly FaxSystem _fax = default!;
+    [Dependency] private readonly IConfigurationManager _config = default!;
+    [Dependency] private readonly StationSystem _station = default!;
+
+    private static readonly Regex StationIdRegex = new(@".*-(\d+)$");
+
+    private const string RandomPrototype = "StationGoals";
```

Comment:
Validate this and maybe make it a CVar.

Reply:
![image](https://github.com/Simple-Station/Einstein-Engines/assets/16548818/c5138fb3-cb26-45d8-8d7c-2f80da89c308)

I apologize, but it's not possible to validate the prototype of a CVar string. I can Validate it, I can make it a CVar, but I can't make it a CVar AND validate it. In the meantime I'm just going to validate it.

Reply:
Make it a CVar

Reply:
Oh my god you edited it to say "Validate it", but I didn't get the push notification that you edited this, so I got a message on my desktop saying "CVar it". So I made it a CVar.
AAAAAAAAAAAAAAAAAAAAAAAAA

Reply:
:trollface:

Reply:
Well it's validated now lol.